source/uwp/sharedObjectModel/lib/pch.h

```diff
@@ -35,7 +35,12 @@
 #include <winrt/base.h>
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.Foundation.Collections.h>
+#ifndef USE_WINUI3
 #include <winrt/AdaptiveCards.ObjectModel.Uwp.h>
+#else
+#define Uwp Winui3
```

Comment:
are you overriding Uwp with the value of Winui3? if so, can you rename Uwp, so it would make more sense?

Reply:
Sure thing. Originally I didn't want to have to touch every lib file, but I'm already moving those now.

Reply:
I changed Uwp to Xaml_OM.